google-cloud-storage/src/main/java/com/google/cloud/storage/StorageException.java

```diff
@@ -73,4 +82,25 @@ public static StorageException translateAndThrow(RetryHelperException ex) {
     BaseServiceException.translate(ex);
     throw new StorageException(UNKNOWN_CODE, ex.getMessage(), ex.getCause());
   }
+
+  /**
+   * Translate IOException to the StorageException that caused the error. This method default to
+   * idempotent always being {@code True}. This method will force retry of the following transient
```

Comment:
true? (primitive)
will force --> forces
or perhaps just will force retry of --> retries